src/utils/mixins/editMixin.js

```diff
@@ -32,12 +32,12 @@ export default {
     },
   },
   methods: {
-    save (event) {
```

Comment:
I wonder why we had `event` here in the first place...?

Reply:
I wondered that too, I did my best to look at all the places it's called, and couldn't find anything that looked like it would break...

I think some pre-vue-query thing might have used it when clicking on a link, to check which element was intended (event.target.closest('a') ... or something...).